common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -431,24 +443,11 @@ describe 'Problem', ->
         expect($('#answer_1_1')).toHaveHtml 'One'
         expect($('#answer_1_2')).toHaveHtml 'Two'
 
-      it 'toggle the show answer button', ->
-        spyOn($, 'postWithPrefix').and.callFake (url, callback) -> callback(answers: {})
-        @problem.show()
-        expect($('.show .show-label')).toHaveText 'Hide Answer'
-        expect(window.SR.readElts).toHaveBeenCalled()
-
-      it 'toggle the show answer button, answers are strings', ->
-        spyOn($, 'postWithPrefix').and.callFake (url, callback) -> callback(answers: '1_1': 'One', '1_2': 'Two')
-        @problem.show()
-        expect($('.show .show-label')).toHaveText 'Hide Answer'
-        expect(window.SR.readElts).toHaveBeenCalledWith ['<p>Answer: One</p>', '<p>Answer: Two</p>']
-
       it 'toggle the show answer button, answers are elements', ->
```

Comment:
Similarly here just change the test point description so that it only references what is tested.

````
it 'sends the answers when elements to the SR element', ->